### ISSUE TITLE
New global responsive grid layout

### ISSUE THREAD

**Issue by robinmetral** [wikitongues/wikitongues-website#7]
Edit the template files and CSS to make the site mobile-first (@media queries for screens larger than X instead of smaller than X)

**Comment by robinmetral** [comment]
Working on this as of today!

@bogreudell I've postponed the idea of rewriting the overall layouts to make them seamlessly responsive with a grid. It would take too long and affect too many other design elements.
Instead, for now, I'm using media queries to make the current layouts display properly on mobile.

I'm running into a few problems with the nav @bogreudell. Do you know if Wordpress has built-in features to easily make a top nav become a hamburger icon?
Otherwise I might have to change our navs, but they are PHP variables. Where yould I edit them? (I'm thinking putting the current two navs into one, and adding a hidden element to toggle the hamburger button).

**Comment by bogreudell** [comment]
@robinmetral I don't believe WP has any built-in functions that would render this. We'll need to compose the markup manually. Another solution for the mobile menu would simply be to have a horizontal scroll. What do you think?

**Comment by robinmetral** [comment]
By horizontal scroll, do you mean that the nav would stay the same as on desktop but overflow with a scroll to the right?
Do you have an example? :slightly_smiling_face:

**Comment by robinmetral** [comment]
@bogreudell Done :heavy_check_mark: 
Now all that remains until we have a basic responsive site, is typography and spacing (which need to become smaller as screen size decreases) :tada:

**Comment by robinmetral** [comment]
*Nav:* hide scroll bar and make logo fixed
